kolibri/plugins/management/assets/src/vue/manage-content-page/wizard-export.vue

```diff
@@ -148,11 +204,21 @@
     margin-right: 0.2em
     margin-bottom: -6px
 
-  .drive-list
-    margin: 2em
-
   .drive-names
-    margin: 0.6em 0
+    padding: 0.6em
+    border: 1px $core-bg-canvas solid
```

Comment:
very minor point, but I would consider removing this border:

* the blocks are effectively distinguished with the padding - and they look great!
* it makes it look like the whole block should be clickable, when in fact only the labels are